SensusService/Probes/User/ScriptRunner.cs

```diff
@@ -187,7 +201,7 @@ public string RandomTriggerWindows
                 if (_randomTriggerWindows.Count == 0)
                     return "";
                 else
-                    return string.Concat(_randomTriggerWindows.Select((window, index) => (index == 0 ? "" : ",") + window.Item1 + "-" + window.Item2));
+                    return string.Concat(_randomTriggerWindows.Select((window, index) => (index == 0 ? "" : ",") + (window.Item1 == window.Item2 ? window.Item1 + "" : window.Item1 + "-" + window.Item2)));    // have to add extra "" in first clause of second ternary conditional, throws string to DateTime conversion error otherwise
```

Comment:
window is a Tuple<DateTime, DateTime> object and doesn't seem to have a toString() method, nor does window.Item1